README.md

```diff
@@ -0,0 +1,9 @@
+Welcome to Alien Invasion!
```

Comment:
Cracking start to the README! A bit of character is always great to see rather than just a dry set of instructions. It is missing a few bits that would make it much better though:

- installation instructions (ie cloning the repo and what commands are available),
- what are the rules for how the pet is fed, exercised, how long it will live, what are the max and min values for things like hunger, fitness etc. You mention all these things and thats great, but how are these things actually done by a user? Some code snippets would be very useful
- how the app is tested,
- what you would do if you had more time
- your name